requirements.txt

```diff
@@ -1,11 +1,11 @@
-fastapi
 fire
-httpx
 mesop
 python-dotenv
 setuptools
-uvicorn
 yfinance
-gradio
-llama-stack>=0.0.50
-llama-stack-client
+llama-stack==0.0.53rc1
```

Comment:
we certainly _cannot_ add all these dependencies just for a single app. we should put a separate set of requirements for the app and ask users to install them.

Reply:
agreed, creating a separate requirement.txt in project dir to reflect this @wukaixingxp  @init27